internal/controller/common/backup_config_util.go

```diff
@@ -16,3 +34,119 @@ func GetConfigSection(config map[string]interface{}, section string) (map[string
 
 	return sectionMap, nil
 }
+
+func GetBackupServicePodList(k8sClient client.Client, name, namespace string) (*corev1.PodList, error) {
+	var podList corev1.PodList
+
+	labelSelector := labels.SelectorFromSet(utils.LabelsForAerospikeBackupService(name))
+	listOps := &client.ListOptions{
+		Namespace: namespace, LabelSelector: labelSelector,
+	}
+
+	if err := k8sClient.List(context.TODO(), &podList, listOps); err != nil {
+		return nil, err
+	}
+
+	return &podList, nil
+}
+
+func ReloadBackupServiceConfigInPods(
+	k8sClient client.Client,
+	log logr.Logger,
+	backupSvc *v1beta1.BackupService,
+) error {
+	if err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+		podList, err := GetBackupServicePodList(k8sClient,
+			backupSvc.Name,
+			backupSvc.Namespace,
+		)
+		if err != nil {
+			return fmt.Errorf("failed to get backup service pod list, error: %v", err)
+		}
+
+		for idx := range podList.Items {
+			pod := podList.Items[idx]
+			annotations := pod.Annotations
+
+			if annotations == nil {
+				annotations = make(map[string]string)
+			}
+
+			count, ok := annotations[v1beta1.ForceRefreshKey]
+			if ok {
+				countInt, _ := strconv.Atoi(count)
+				annotations[v1beta1.ForceRefreshKey] = fmt.Sprintf("%d", countInt+1)
+			} else {
```

Comment:
TODO: only 1 annotation is enough.

Reply:
Kept timestamp annotation as per our discussion